ISSUE_TEMPLATE.md

```diff
@@ -6,7 +8,14 @@
 
 ### Additional Information
 
+### Environment
+- OSC version/build: 
```

Comment:
Follow the capitalization used in other places: OSC Version/Build, Cloud Provider Info, Plugin Info